### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Winky1st** [Guilouz/KlipperScreen-Flsun-Speeder-Pad#34]
**Describe the bug**
After downloading v0.3.8-3 few minutes back Speederpad has gone black screen

**To Reproduce**
1. download v0.3.8-3
2. there you go

thankyou

**Logfile**
attached
[KlipperScreen.log](https://github.com/Guilouz/KlipperScreen-Flsun-Speeder-Pad/files/14067554/KlipperScreen.log)

**Comment by Guilouz** [comment]
Just pushed a fix for that. Can you update KlipperScreen from Update Manager and tell me if it's ok.

**Comment by Winky1st** [comment]
100 working again - Fixed, 
Thank you very much Cyril . Cheers.

**Comment by Guilouz** [comment]
Perfect!

**Comment by RHYANGHA** [comment]
Hello, it seems that I have had the same problem with both of my printers. I updated version 3.8-17 and both were left with black klipperscreen. 
The klipperscreen.conf file of one of them was left this way, I tried replacing the installation file but there was no solution.
![IMG-20240227-WA0003](https://github.com/Guilouz/KlipperScreen-Flsun-Speeder-Pad/assets/157091261/956a1a79-6825-4b68-ba2c-120dd9281f33)
![IMG-20240227-WA0004](https://github.com/Guilouz/KlipperScreen-Flsun-Speeder-Pad/assets/157091261/385c13c0-d653-4acd-83ab-1704a354f026)
[KlipperScreen.log](https://github.com/Guilouz/KlipperScreen-Flsun-Speeder-Pad/files/14422064/KlipperScreen.log)
I would be eternally grateful for your help. Have a nice day
